restapi/stories.go

```diff
@@ -32,30 +32,148 @@ func init() {
 	rand.Seed(time.Now().UTC().UnixNano())
 }
 
+type TaggedStory struct {
+	models.Story
+	Tags []string `json:"tags"`
```

Comment:
plz update the typescript type in `types.tsx` to reflect the new story type being returned.